circuits/cpp/barretenberg/cpp/src/barretenberg/ecc/curves/secp256k1/secp256k1.cpp

```diff
@@ -4,7 +4,9 @@ namespace secp256k1 {
 namespace {
 
 constexpr size_t max_num_generators = 1 << 10;
+// NOLINTNEXTLINE TODO(@zac-williamson) get rid of need for these static variables in Pedersen refactor!
```

Comment:
Same comment above about issue/PR relating to pedersen refactor

Reply:
done